*** Drill - Exercises in Bayes' Rule

A diagnostic test has a 98% probability of giving a positive result when applied to a person suffering from Thripshaw's Disease, and 10% probability of giving a (false) positive when applied to a non-sufferer. It is estimated that 0.5 % of the population are sufferers. Suppose that the test is now administered to a person whose disease status is unknown. Calculate the probability that the test will:

1. Be positive

In [1]:
positive_result = (.98 *.005) + (.1 * (.995))
print(positive_result)

0.1044


A: There is 0.1044 or 10.44% chance the test will test.

2. Correctly diagnose a sufferer of Thripshaw's

A: We were already told this test is 98% accurate when testing against those with the disease

3. Correctly identify a non-sufferer of Thripshaw's

In [8]:
accurate_result = (1 - .1)
print(accurate_result)

0.9


A: 90% of the time the test properly gives negative results to patients without the disease.
10% of the time there is a false positive.

4. Misclassify the person

In [7]:
bad_result = (.02 * .005)+ (.1 * .995)
print(bad_result)

0.09960000000000001


A: Needed to add the probability of false negatives and false positives to equal 0.09960000000000001 or 9.96% chance for a bad diagnosis from a single result.